### PR TITLE
feat: add `DIDSet` and `DIDDelete` transactions

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Title says it all.

### Context of Change

https://github.com/XRPLF/rippled/pull/4636
https://github.com/XRPLF/XRPL-Standards/discussions/100

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After

image

image

image

## Test Plan

I added tests. CI passes.

### CODE REVIEW

File: src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      {URI && (
+        <div className="uri">
+          <span className="label">{t('uri')}: </span>
+          <span className="lower">{convertHexToString(URI)}</span>
+        </div>
+      )}
+      {URI && (
+        <div className="did-document">
+          <span className="label">{t('did_document')}: </span>
+          <span className="lower">{convertHexToString(DIDDocument)}</span>
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx:22]: Why is the class name `lower`?

Comment (mvadari) [reply src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx:22]: Short for "lowercase" - though I guess I really meant case-sensitive

Comment (ckniffen) [reply src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx:22]: That is what i figured but it has the same styling as `.account` which isn't only lower case so I caught it.

Comment (mvadari) [reply src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx:22]: I can change it to `case-sensitive` or `no-upper`

Comment (ckniffen) [reply src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx:22]: `case-sensitive` is my vote.

**Review by jonathanlei — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

**Pull request merged**